The pkg-config files of some conda-forge packages appear to have
issues.  HDF5 is not found anywhere, and LibTIFF is similarly affected
on Windows.  For now, use CMake configuration files instead.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -441,21 +441,10 @@
 if(CBF_WITH_CQRLIB)
   fetchcontent_makeavailable(cqrlib)
 else()
-  # Should use pkg-config or some such once released CQRlib provides
-  # those mechanisms.
-  find_library(CQRLIB_LIBRARY CQRlib)
-  find_path(CQRLIB_INCLUDE_DIR cqrlib.h)
-
-  add_library(CQR UNKNOWN IMPORTED)
-  set_target_properties(CQR PROPERTIES
-    IMPORTED_LOCATION "${CQRLIB_LIBRARY}"
-    INTERFACE_INCLUDE_DIRECTORIES "${CQRLIB_INCLUDE_DIR}"
-    INTERFACE_LINK_LIBRARIES "${libm}")
-
   # Alternatively, find CQRlib using its CMake Config-file.  This will
   # not work in the general case until CQRlib has CMake support.
-  #find_package(CQRlib CONFIG REQUIRED)
-  #add_library(CQR ALIAS CQR::CQR)
+  find_package(CQRlib CONFIG REQUIRED)
+  add_library(CQR ALIAS CQR::CQR)
 endif()
 
 
@@ -523,10 +512,10 @@
   endif()
 
 else()
-  find_package(PkgConfig REQUIRED)
-  pkg_check_modules(hdf5 REQUIRED IMPORTED_TARGET hdf5>=1.10)
+  find_package(HDF5 REQUIRED)
+  add_library(HDF5 ALIAS hdf5::hdf5)
   find_program(_h5dump_executable h5dump REQUIRED)
-  set(_hdf5_target PkgConfig::hdf5)
+  set(_hdf5_target hdf5::hdf5)
 endif()
 
 
@@ -599,9 +588,8 @@
   # Use pkg-config, with the intent of eventually switching to
   # cmake_pkg_config(), because e.g. Debian does not ship CMake config
   # files in their dev packages.
-  find_package(PkgConfig REQUIRED)
-  pkg_check_modules(tiff REQUIRED IMPORTED_TARGET libtiff-4>=4.0.10)
-  add_library(tiff ALIAS PkgConfig::tiff)
+  find_package(TIFF REQUIRED)
+  add_library(tiff ALIAS TIFF::TIFF)
 endif()
 
 
